lib/actions/ui.js

```diff
@@ -396,3 +399,14 @@ export function setRouteViewerFilter(filter) {
     }
   }
 }
+
+/**
+ * Start over by setting the initial URL and resetting (reloading) the page.
+ */
+export function startOverFromInitialUrl() {
```

Comment:
Did you check if there's a function that does this already from the sidebar's "start over" button?

Reply:
This code is slightly different from the "Start Over" buttons that clear out the original query params used to load the page. We need to preserve those original query params. I'll add a comment regarding that need, thanks for pointing this out.